packages/mattermost-redux/src/utils/post_list.test.js

```diff
@@ -1526,75 +1524,76 @@ describe('combineUserActivitySystemPost', () => {
                 {postType: PostTypes.LEAVE_CHANNEL, userIds: ['user_id_1', 'user_id_2']},
             ],
         };
-        assert.deepEqual(combineUserActivitySystemPost([postLeaveChannel1, postLeaveChannel2, postLeaveTeam1, postLeaveTeam2]), out3);
+        assert.deepEqual(combineUserActivitySystemPost([postLeaveChannel2, postLeaveChannel1, postLeaveTeam2, postLeaveTeam1]), out3);
 
         const out4 = {
-            allUserIds: ['removed_user_id_1', 'removed_user_id_2', 'user_id_1', 'user_id_2'],
+            allUserIds: ['user_id_1', 'user_id_2', 'removed_user_id_1', 'removed_user_id_2', ''],
             allUsernames: [],
             messageData: [
                 {postType: PostTypes.REMOVE_FROM_TEAM, userIds: ['user_id_1', 'user_id_2']},
-                {actorId: 'user_id_1', postType: PostTypes.REMOVE_FROM_CHANNEL, userIds: ['removed_user_id_1', 'removed_user_id_2']},
+                {actorId: '', postType: PostTypes.REMOVE_FROM_CHANNEL, userIds: ['removed_user_id_1', 'removed_user_id_2']},
             ],
         };
-        assert.deepEqual(combineUserActivitySystemPost([postRemoveFromChannel1, postRemoveFromChannel2, postRemoveFromTeam1, postRemoveFromTeam2]), out4);
+        assert.deepEqual(combineUserActivitySystemPost([postRemoveFromChannel2, postRemoveFromChannel1, postRemoveFromTeam2, postRemoveFromTeam1]), out4);
 
         const out5 = {
-            allUserIds: ['added_user_id_1', 'added_user_id_2', 'user_id_1', 'user_id_2', 'removed_user_id_1', 'removed_user_id_2'],
+            allUserIds: ['removed_user_id_1', 'removed_user_id_2', '', 'user_id_1', 'user_id_2', 'added_user_id_1', 'added_user_id_2'],
```

Comment:
I noticed a few "undefined" values in the test results when I was running those locally. I think it might be because of the empty ID in here which doesn't seem like it should be there

Reply:
I will probably have to add tests and  change these tests , I will check them one by the one